### DISCUSSION TITLE
Bug report:The Repos API returns the same value for "stargazers_count" and "watchers_count"

### DISCUSSION THREAD

**Discussion by noobyu6** [community/community#8135] [Projects and Issues]
Hi,
    Recently, when I use "GET /orgs/{org}/repos" to get  repository information, then I find stargazers_count's value and watchers_count's are the same. I think it's a bug. Link is "https://docs.github.com/en/rest/reference/repos#list-organization-repositories". my shell command is "curl -H 'Accept: application/vnd.github.v3+json;charset=utf-8' https://api.github.com/repos/brianfrankcooper/YCSB". I hope to receive your reponse, thanks!

**Comment by jiacai2050** [comment]
It's not a bug.
- https://github.community/t/bug-watchers-always-same-as-stargazers-count-in-api-user-starred/14224/2?u=jiacai2050
